client/signup/jetpack-connect/plans.jsx

```diff
@@ -58,12 +58,11 @@ const Plans = React.createClass( {
 	},
 
 	componentWillReceiveProps( props ) {
-		const selectedSite = props.sites.getSelectedSite();
```

Comment:
>Is there any particular reason to use the old props in `componentWillReceiveProps` instead of the new ones (as it was before)?

Seconded. You probably want the treat the `props` argument as the most up to date. You could use `componentDidUpdate` instead if you'd rather use `this.props` (since this calls to many other functions on the component).